### ISSUE TITLE
Letter-spacing style props caused Text component to breaks at incorrect positions

### ISSUE THREAD

**Issue by sbycrosz** [facebook/react-native#29958]
## Description

After the Android 11 release, some of our users noticed that some texts are rendered wrongly

![Screenshot_20200916-162432](https://user-images.githubusercontent.com/4867178/93314056-de9fe080-f83b-11ea-8dc5-2016db1ea741.png)

It renders properly after removing `letterSpacing: -0.8` style from the text components

![Screenshot_20200916-162448](https://user-images.githubusercontent.com/4867178/93314061-dfd10d80-f83b-11ea-859c-2988b53a44aa.png)

Additionaly, it renders properly on iOS and Android 10 devices

## React Native version:

```
System:
    OS: macOS 10.15.4
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Memory: 821.89 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.12.0 - ~/.nvm/versions/node/v12.12.0/bin/node
    Yarn: 1.7.0 - /usr/local/bin/yarn
    npm: 6.11.3 - ~/.nvm/versions/node/v12.12.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 19, 22, 23, 24, 25, 26, 28
      Build Tools: 23.0.1, 23.0.2, 23.0.3, 24.0.2, 25.0.0, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 27.0.3, 28.0.3
      System Images: android-28 | Google Play Intel x86 Atom, android-30 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 11.4/11E146 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: ^0.61.5 => 0.61.5
✨  Done in 6.82s.
```

## Steps To Reproduce
1. Create Text component with letterSpacing
2. Run the code on Android 11 device (edit: It's all good on Android 11 emulator, but not on real device)
3. Text will breaks at incorrect positions

## Expected Results
Text should render properly (pic2)

**Comment by htduy262** [comment]
I got the same issue with letter spacing in Android 11 (both emulator and device).

Expected
![image](https://user-images.githubusercontent.com/14125876/97958690-4d330080-1de0-11eb-9727-207af603eca5.png)

Actual
![image](https://user-images.githubusercontent.com/14125876/97957773-718ddd80-1dde-11eb-934d-885150ab3c66.png)

I've upgraded the latest version of React Native 0.63.3 but still got the issue.
My workaround solution is round the letterSpacing value to 2 decimals (-0.XX).

Any suggestions?
Thanks for any help.
